source/material_model/simpler.cc

```diff
@@ -65,6 +73,15 @@ namespace aspect
           out.specific_heat[i] = reference_specific_heat;
           out.thermal_conductivities[i] = k_value;
           out.compressibilities[i] = 0.0;
+
+          derivatives->dviscosities_du[i] = 0.0;
```

Comment:
What is meant is that the function also needs to successfully run if the caller is not actually interested in derivatives. In that case, `derivatives` is just a NULL pointer and you should not dereference it. So you need a
```
  // see if we need to compute derivatives
  if (derivatives != 0)
   {
     ...
   }
```
block around lines 77 to 84.